lte/cloud/go/services/subscriberdb/obsidian/models/swagger.v1.yml

```diff
@@ -128,6 +128,25 @@ paths:
         default:
           $ref: './orc8r-swagger-common.yml#/responses/UnexpectedError'
 
+  /lte/{network_id}/subscribers_v2?verbose=foorbar:
+    get:
+      summary: List names of subscribers in the network
+      tags:
+        - Subscribers
+      parameters:
+        - $ref: './orc8r-swagger-common.yml#/parameters/network_id'
+        - $ref: './orc8r-swagger-common.yml#/parameters/page_size'
+        - $ref: './orc8r-swagger-common.yml#/parameters/page_token'
+      responses:
+        '200':
+          description: List of names of subscribers in the network
+          schema:
+            type: array
+            items:
+              $ref: './lte-policydb-swagger.yml#/parameters/subscriber_id'
+        default:
+          $ref: './orc8r-swagger-common.yml#/responses/UnexpectedError'
+
```

Comment:
You should be able to just edit the block above starting from line 87. Instead of appending `?verbose=foorbar` to the path, add a new `parameter` entry under `get`:

```
parameters:
  - in: query
    name: verbose
    .....
```

Reply:
By default you'll find the help for Swagger 3.0, but we're using Swagger 2.0, see here: https://swagger.io/specification/v2/

Reply:
Makes sense, thanks!